drv/stm32h7-sprot-server/src/main.rs

```diff
@@ -193,8 +148,8 @@ pub struct Io<S: SpiServer> {
 
 pub struct ServerImpl<S: SpiServer> {
     io: Io<S>,
-    tx_buf: [u8; BUF_SIZE],
-    rx_buf: [u8; BUF_SIZE],
+    tx_buf: [u8; MAX_REQUEST_SIZE],
```

Comment:
Similar suggestion as above: it would be more stack-friendly to use `mutable_statics!` and put them into static memory.

Reply:
Great call. Fixed in [3ddb01b](https://github.com/oxidecomputer/hubris/pull/1279/commits/3ddb01b536d37432e9c62a391d4b708a2630ce23)